### ISSUE TITLE
Wrong format when generating empty scopes map

### ISSUE THREAD

**Issue by JohanWinther** [biocad/openapi3#63]
Having an empty [`_oAuth2Scopes`](https://hackage.haskell.org/package/openapi3-3.2.2/docs/Data-OpenApi.html#v:_oAuth2Scopes) map generates the following object which is invalid:
```json
{
  "securitySchemes": {
    "OAuth2": {
      "description": "Description",
      "flows": {
        "clientCredentials": {
          "tokenUrl": "https://example.com"
        }
      },
      "type": "oauth2"
    }
  }
}
```

To make it valid the `scopes` property needs to be added, but with an empty object:

```json
{
  "securitySchemes": {
    "OAuth2": {
      "description": "Description",
      "flows": {
        "clientCredentials": {
          "tokenUrl": "https://example.com",
          "scopes": {} // Added
        }
      },
      "type": "oauth2"
    }
  }
}
```

**Comment by JohanWinther** [comment]
I have found a fix and will submit a pull request soon.

**Comment by maksbotan** [comment]
Fixed with #65, thanks!
